client/dist/index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Space work reviews</title>
+  <style>
+    .proxy-container {
+      margin: auto;
+      width: 90%;
+      max-width: 1200px;
+    }
+
+    .proxy-container-columns {
+      display: grid;
+      grid-template-columns: 60% 40%;
+      grid-gap: 15px;
+    }
+
+    @media screen and (max-width: 1000px) {
+      .proxy-container-columns {
+        display: block;
+      }
+    }
+
+    .col-2 {
+      height: 100%;
+    }
+
+    #location-service > div > h3 {
+      padding-top: 50px;
+    }
+  </style>
+  <link rel="stylesheet" type="text/html" href="reset.css">
+  <link href='https://api.mapbox.com/mapbox-gl-js/v2.0.1/mapbox-gl.css' rel='stylesheet' />
+</head>
+<body>
+  <div class="proxy-container">
+    <div class="proxy-container-columns">
+      <div class="col-1">
+        <div id="workspace-header"></div>
+        <div id="photos-slider"></div>
+        <div id="workspace-description"></div>
+        <div id="workspaces"></div>
+        <div id="location-service"></div>
+        <div id="reviews"></div>
+        <div id="nearby"></div>
+      </div>
+      <div class="col-2">
+        <div id="contact-service"></div>
+      </div>
+    </div>
+  </div>
+</body>
+<script crossorigin src="https://unpkg.com/react@17/umd/react.development.js"></script>
+<script crossorigin src="https://unpkg.com/react-dom@17/umd/react-dom.development.js"></script>
+<script crossorigin src="https://cdnjs.cloudflare.com/ajax/libs/react-router-dom/5.2.0/react-router-dom.min.js"></script>
+<script crossorigin src="https://cdnjs.cloudflare.com/ajax/libs/axios/0.21.1/axios.min.js"></script>
+<script src="https://kit.fontawesome.com/bb32e62705.js" crossorigin="anonymous"></script>
+<!-- Photos and header, and description, port 6000s -->
+<script src="https://rpt25-photos-service.s3-us-west-1.amazonaws.com/js/photos-slider.js"></script>
+<script src="https://rpt25-photos-service.s3-us-west-1.amazonaws.com/js/workspace-description.js"></script>
+<!-- Workspaces, port 4000 -->
+<script src="https://workspaces-service.s3-us-west-1.amazonaws.com/dist/workspaces.js"></script>
+<!-- Location and contact, 3000s -->
+<script src="https://fec-location-service.s3.us-east-2.amazonaws.com/main.js"></script>
+<script src="https://fec-contact-service.s3.us-east-2.amazonaws.com/main.js"></script>
+<!-- Nearby and reviews, port 5000s -->
+<script src="https://spacework-bundles.s3-us-west-1.amazonaws.com/dev-nearby.js"></script>
+<script src="http://ec2-184-169-176-177.us-west-1.compute.amazonaws.com/reviews.js"></script>
```

Comment:
This lives on s3 as well: https://spacework-bundles.s3-us-west-1.amazonaws.com/reviews.js

Reply:
Ah, good catch, thanks